cglm.pc.in

```diff
@@ -1,11 +1,11 @@
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-libdir=@libdir@
-includedir=@includedir@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix="${prefix}"
+libdir="${exec_prefix}/lib"
+includedir="${prefix}/include"
```

Comment:
```suggestion
prefix=@CMAKE_INSTALL_FULL_PREFIX@
exec_prefix="${prefix}"
libdir=@CMAKE_INSTALL_FULL_LIBDIR@
includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
```

Reply:
That results in non-idiomatic pkg-config files that don't handle relocation.

> The variables in pkg-config files are expected to contain interpolations (e.g. `completiondir=${datadir}/bash-completion`) so that projects can install files relative their own directories instead of the ones of the project whose variables they use (see https://www.bassi.io/articles/2018/03/15/pkg-config-and-paths/). This means you cannot just use the `CMAKE_INSTALL_FULL_<dir>` variables and need to construct the paths yourself.